sandbox/libs/authn/src/main/java/org/opensearch/authn/realm/InternalRealm.java

```diff
@@ -84,4 +85,114 @@ protected AuthenticationInfo doGetAuthenticationInfo(AuthenticationToken token)
         // Don't know what to do with this token
         throw new CredentialsException();
     }
+
+    // TODO: Expose all the operations below as a rest API
+
+    /**
+     * Creates a subject in an in-memory data store
+     * @param subject to be created. It should be passed in {@link InternalSubject}
+     */
+    public void createSubject(InternalSubject subject) {
+        String primaryPrincipal = subject.getPrimaryPrincipal();
+
+        // TODO: should we update if an object already exists with same principal.
+        // If so, it should be handled in updateSubject
+        if (this.internalSubjects.containsKey(primaryPrincipal)) return;
```

Comment:
```suggestion
        if (this.internalSubjects.containsKey(primaryPrincipal)) {
           return;
        }
```

I'm a stickler about always using brackets, mind making this change? (Its in a couple of places)